bioblend/galaxyclient.py

```diff
@@ -63,10 +63,8 @@ def __init__(self, url, key=None, email=None, password=None, verify=True, timeou
             self._key = None
             self.email = email
             self.password = password
-        self.json_headers = {
-            'Content-Type': 'application/json',
-            'x-api-key': self.key,
-        }
+        self.json_headers = {'Content-Type': 'application/json'}
```

Comment:
Good catch! Can you please add a comment between these 2 lines to explain the reason why `self.json_headers` need to exist before `self.key` is accessed?